infra/charts/turing/templates/virtualservice-mlp.yaml

```diff
@@ -0,0 +1,88 @@
+{{- if .Values.tags.mlp -}}
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: {{ template "turing.fullname" . }}-gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 80
+        name: http
+        protocol: HTTP
+      hosts:
+        - "*"
+
+---
+apiVersion: networking.istio.io/v1beta1
+kind: VirtualService
+metadata:
+  name: {{ template "turing.fullname" . }}-virtual-service
+spec:
+  hosts:
+    - "*"
+  gateways:
+    - {{ template "turing.fullname" . }}-gateway
+  http:
+    - match:
+        - uri:
+            prefix: "/api/turing/"
+        - uri:
+            prefix: "/api/turing"
+      rewrite:
+        uri: "/"
+      route:
+        - destination:
+            host: {{ template "turing.fullname" . }}
+            port:
+              number: 8080
+    - match:
+        - uri:
+            prefix: "/api/merlin/"
```

Comment:
Is this necessary? `/api/merlin` is a subset of `/api/merlin/`

Reply:
There is an issue with istio https://github.com/istio/istio/issues/8076#issuecomment-427057691 on this.

Reply:
This is hilarous, could you add a comment inside with this link to show why it was done this way?